osu.Game/Screens/Play/Player.cs

```diff
@@ -420,10 +426,14 @@ private void updateSampleDisabledState()
             samplePlaybackDisabled.Value = DrawableRuleset.FrameStableClock.IsCatchingUp.Value || GameplayClockContainer.GameplayClock.IsPaused.Value;
         }
 
-        private void updatePauseOnFocusLostState() =>
-            HUDOverlay.HoldToQuit.PauseOnFocusLost = PauseOnFocusLost
-                                                     && !DrawableRuleset.HasReplayLoaded.Value
-                                                     && !breakTracker.IsBreakTime.Value;
+        private void updatePauseOnFocusLostState()
+        {
+            if (!PauseOnFocusLost || breakTracker.IsBreakTime.Value)
```

Comment:
Why is the "replay loaded" check gone now? I don't recall requesting removing it.

Reply:
It's not something to care about anymore given that this calls `Pause()` directly instead of `performUserRequestedExit` previously (where if a replay is loaded it'll exit from player immediately)

Reply:
Half an answer again... `canPause` inside of `Pause()` checks `HasReplayLoaded` anyway is the actual complete answer.

Reply:
<img width="434" alt="image" src="https://user-images.githubusercontent.com/22781491/107126059-a5d1d800-68be-11eb-9099-4fcb65f2afd0.png">
I've put it in the description thought you would read it, but sure will try answer completely next time.